### PR TITLE
French Translation: Challenge Uncomment HTML

### PR DESCRIPTION
**Pull request by SonyaMoisset**
French Translation: Challenge-Uncomment-HTML.md

### CODE REVIEW

**Comment by alayek** [comment]
# LGTM :+1: 

I am waiting for someone who speaks French to review it.

**Comment by theflametrooper** [comment]
@fnonga Could you verify this translation?

File: fr.lang/Challenge-Décommenter-HTML.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Les commentaires permettent au code d'être ignoré. Dans le language HTML, les commentaires sont faits en entourant le code que vous souhaitez ignorer avec `<!--` et `-->`.
```
Comment (yaax) [inline fr.lang/Challenge-Décommenter-HTML.md]: " Dans le langage HTML"  (no U in "langage")

File: fr.lang/Challenge-Décommenter-HTML.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+`<! - Ce texte n'apparaîtra pas puisqu'il est un commentaire ->`
+
+Vous pouvez facilement décommentez du code HTML en supprimant les éléments de commentaire.
```
Comment (yaax) [inline fr.lang/Challenge-Décommenter-HTML.md]: décommente**r**

File: fr.lang/Challenge-Décommenter-HTML.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
+***Remarque:*** Soyez prudent quand vous décommentez du code HTML et penser à bien enlever les éléments du début et de la fin de votre commentaire!
```
Comment (yaax) [inline fr.lang/Challenge-Décommenter-HTML.md]: et pense**z**

File: fr.lang/Challenge-Décommenter-HTML.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+
+***Note:*** Les commentaires peuvent être faits différemment dans d'autres languages de programmation, comme le JavaScript, mais l'idée générale d'enlever tout ce qui représente un commentaire s'applique.
```
Comment (yaax) [inline fr.lang/Challenge-Décommenter-HTML.md]: **langages**

**Comment by theflametrooper** [comment]
@SonyaMoisset Could you update the pr with the changes @yaax made.

**Pull request merged**